node/engine/store/store.go

```diff
@@ -15,6 +15,7 @@ const (
 	ErrNoSuchObjective = types.ConstError("store: no such objective")
 	ErrNoSuchChannel   = types.ConstError("store: failed to find required channel data")
 	ErrLoadVouchers    = types.ConstError("store: could not load vouchers")
+	lastBlockSeenKey   = "lastBlockSeen"
```

Comment:
Another instance of "block" vs "blocknum"

Reply:
Changed in https://github.com/statechannels/go-nitro/pull/1581/commits/5d4a6161745e089a900cf5dc42e642b1116a3a44